.github/dependabot.yml

```diff
@@ -1,5 +1,15 @@
 version: 2
 updates:
+  - package-ecosystem: gomod
+    directory: /
```

Comment:
That's a great question -- you all know more about Go dependencies than I do, and if you don't know then we may just have to try this out and see what happens. My rough understanding (if it works like it does for JS/NPM), then the `directory: /` here means it's responsible _only_ for updating the top-level `go.mod` file and not any of the ones in subdirectories (i.e. the backends) _unless_ a change to the top-level `go.mod` file requires a change in the subdirectories. That's how it works in JS but I don't know if there are any differences in the behavior for Go.

Reply:
Our repo pretty much always requires changes to all module directories, which is handled manually via our `make syncdeps` command. We can't exclude backends, because their `go.mod` files will immediately be out of sync. I don't know if dependabot can handle that, or if we ned to add a followup command to clean up after the update.

Reply:
Interesting, yeah, I think in that case we'll just have to merge this and see what happens, and if additional steps are needed then that probably requires the creation of a custom workflow that runs additional commands based on the Dependabot PR.